### ISSUE TITLE
Screen sharing broke after regression

### ISSUE THREAD

**Issue by Aqa-Ib** [hyprwm/xdg-desktop-portal-hyprland#225]
No screen sharing on OBS, webcord or vesktop.
Commit 12af8413417d5ad693348ee5b140a426471b0be6 broke it.

Cheers

Ref: https://github.com/hyprwm/Hyprland/issues/6396

**Comment by Aqa-Ib** [comment]
That commit breaks screen sharing on hyprland 0.41.0, but does not break hyprland 0.40.0.

**Comment by Aqa-Ib** [comment]
```
g_pPortalManager->addTimer({std::clamp(MSTILNEXTREFRESH - 1.0 /* safezone */, 1.0, 1000.0), [pSession]() { g_pPortalManager->m_sPortals.screencopy->startFrameCopy(pSession); }});
```

Facts on my end:
- When I set the minimum value of the clamp to 6, the portal rarely fails with "Out of buffers" .
- When I set it to 100, it never fails. But then, the screen capture is slow as before the commit that broke it.

Two things that I don't understand:
- Why does it fails with "Out of buffers" ?
- MSTILNEXTREFRESH seems useless to me because it always returns a value inferior to 1, so it will get clamped. What is the point then ?
